js/script.js

```diff
@@ -0,0 +1,52 @@
+function GithubInteractor(token){
+  this.token = token;
+}
+
+var interactor = new GithubInteractor('b11c7da4c212ae70feae013955c79d4c4014f487')
```

Comment:
If you want this off of your github history, you'll have to do something else.

5 seconds googling, I don't know if this will do the trick.
http://stackoverflow.com/questions/448919/how-can-i-remove-a-commit-on-github

`git log --decorate --all --graph` is the 'prettiest' way to look at your commit history in the terminal.  All intermediary commits get pushed when you do a `git push` unless you do something called a 'squash commit' that squeezes all your intermediary commits into one that I thin is a behavior you might be interested in.